services/opg.digital.metrics/index.md

```diff
@@ -0,0 +1,14 @@
+---
+name: opg.digital.metrics
+summary: |
+  OPG's Metrics system
+owners:
+    - POAS
```

Comment:
I think this is causing the build issue, change to `mrlpa` maybe

Reply:
Yep, owners is the team/user name not the domain. 

There's a separate `domain: POAS` bit of frontmatter for flagging domains.